docker/registry/all-ai-base/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=ghcr.io/deephaven/all-ai-base:latest
+deephaven.registry.imageId=ghcr.io/deephaven/all-ai-base@sha256:6e6d0056d60391409800698bc211ad8b59c5a962c291902da5c93d00243a9a8f
```

Comment:
I think we'll need to bump this to the latest - and update the requirements.txt file.

Reply:
You may also want to update the logic in `docker/registry/scripts/update-requirements.sh` to ensure this image gets regularly bumped

Reply:
done.